src/epcpm/cli/sunspectostaticmodbus.py

```diff
@@ -28,37 +28,57 @@ def map_type_uuid(input, sunspec_types, staticmodbus_types):
     raise ValueError("Error: failure to map type")
 
 
-def create_data_object(
-    input_sunspec_data, scale_factor_uuid_map, sunspec_types, staticmodbus_types
+def create_data_objects(
+    input_sunspec_csv, scale_factor_uuid_map, sunspec_types, staticmodbus_types
 ):
-    # Determine the type of data object to create and then create the object and return it.
-    if input_sunspec_data["class_name"] == epcpm.sunspecmodel.DataPoint.__name__:
-        return create_function_data(
-            input_sunspec_data, scale_factor_uuid_map, sunspec_types, staticmodbus_types
-        )
-    elif (
-        input_sunspec_data["class_name"]
-        == epcpm.sunspecmodel.DataPointBitfield.__name__
-    ):
-        return create_function_data_bitfield(
-            input_sunspec_data, sunspec_types, staticmodbus_types
-        )
-    elif (
-        input_sunspec_data["class_name"]
-        == epcpm.sunspecmodel.TableRepeatingBlockReferenceDataPointReference.__name__
-    ):
-        return create_function_data(
-            input_sunspec_data, scale_factor_uuid_map, sunspec_types, staticmodbus_types
-        )
-    else:
-        raise ValueError(
-            f"Unsupported class_name '{input_sunspec_data['class_name']}' in create_data"
-        )
+    # Create root object to store the static modbus model.
```

Comment:
Can you create docstrings that follow the google standard? And for all the methods?

Reply:
Added docstrings in 73afd44b3cf2d463467b95ab4664543c5ee732e7